enos/modules/aws_boundary/boundary-instances.tf

```diff
@@ -26,6 +26,11 @@ resource "aws_instance" "controller" {
     encrypted   = true
   }
 
+  metadata_options {
+    http_endpoint = "enabled"
+    http_tokens   = "required"
```

Comment:
We may need to add this some of the other modules?
- aws_rdp_domain_controller
- aws_rdp_member_server
- aws_rdp_member_server_with_worker
- aws_windows_client